recipes/sip/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "4.18" %}
+
+package:
+  name: sip
+  version: {{ version }}
+
+source:
+  fn:   sip-{{ version }}.tar.gz  # [unix]
+  url:  http://sourceforge.net/projects/pyqt/files/sip/sip-{{ version }}/sip-{{ version }}.tar.gz  # [unix]
+  sha1: 6704854bc684de3b76c9db61ce54a74e7de5cf45  # [unix]
+  fn:   sip-{{ version }}.zip  # [win]
+  url:  http://sourceforge.net/projects/pyqt/files/sip/sip-{{ version }}/sip-{{ version }}.zip  # [win]
+  sha1: afde507fbee85a1aeb5fcb32e897bd3ea2bc48f5  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - python.app  # [osx]
+    - gcc  # [linux]
```

Comment:
Could we please drop `gcc`? We have an equivalent version of the compiler in the docker image.